html/src/makeHTMLDriver.ts

```diff
@@ -42,9 +45,8 @@ export function makeHTMLDriver(
   function htmlDriver(vnode$: Stream<VNode>, name: string): HTMLSource {
     const html$ = vnode$.map(toHTML);
     html$.addListener({
-      next: effect || noop,
```

Comment:
Any reason why you removed the fallback?

Reply:
Is the fallback need as effect is required?